consensus/istanbul/uptime/uptime.go

```diff
@@ -30,20 +30,21 @@ type Uptime struct {
 // UptimeEntry contains the uptime score of a validator during an epoch as well as the
 // last block they signed on
 type UptimeEntry struct {
-	ScoreTally      uint64
+	// Numbers of blocks validator is considered UP within monitored window
+	UpBlocks        uint64
 	LastSignedBlock uint64
 }
 
 func (u *UptimeEntry) String() string {
-	return fmt.Sprintf("UptimeEntry { scoreTally: %v, lastBlock: %v}", u.ScoreTally, u.LastSignedBlock)
+	return fmt.Sprintf("UptimeEntry { upBlocks: %v, lastBlock: %v}", u.UpBlocks, u.LastSignedBlock)
 }
 
-// GetUptimeMonitoringWindow retrieves the range [first block, last block] where uptime is to be monitored
-// for a give epoch. The range is inclusive.
+// MonitoringWindow retrieves the block window where uptime is to be monitored
+// for a given epoch. The range is inclusive.
```

Comment:
Could drop mention of inclusivity, since Window itself is inclusive. But fine to leave it here.